packages/core/src/linker/component_factory_resolver.ts

```diff
@@ -44,6 +44,9 @@ class _NullComponentFactoryResolver implements ComponentFactoryResolver {
  * @see <live-example name="dynamic-component-loader" noDownload></live-example>
 of the code in this cookbook
  * @publicApi
+ *
+ * @deprecated With Ivy, there is no need to resolve Component factories. No replacement is needed,
+it's possible to use Component Types directly.
```

Comment:
```suggestion
 * @deprecated Angular no longer requires component factories to dynamically
 * create components. All dynamic component creation APIs now directly accept a
 * component class.
```
nit: more active voice (here and elsewhere)

Reply:
Thanks for the feedback Jeremy! I've applied the necessary changes.